scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -447,6 +449,39 @@ def create_branch(
             target_branch, current_release_version_number))
 
 
+def get_release_summary_lines():
+    """Returns the lines from release summary file. It checks whether
+    incorrect email is present or ordering of sections is invalid.
+    In either case, the user will be asked to update the release
+    summary file and the lines will be re-read.
+    """
+    is_invalid_email_present = True
+    is_ordering_invalid = True
+    while is_invalid_email_present or is_ordering_invalid:
+        release_summary_file = python_utils.open_file(
+            release_constants.RELEASE_SUMMARY_FILEPATH, 'r')
+        release_summary_lines = release_summary_file.readlines()
+        is_invalid_email_present = any(
+            release_constants.INVALID_EMAIL_SUFFIX in line
+            for line in release_summary_lines)
+        if is_invalid_email_present:
+            common.ask_user_to_confirm(
+                'The release summary file contains emails of the form: %s '
+                'Please replace them with correct emails.' % (
+                    release_constants.INVALID_EMAIL_SUFFIX))
+        is_ordering_invalid = not(
+            check_ordering_of_sections(release_summary_lines))
```

Comment:
Perhaps call this function: is_order_of_sections_valid()

Reply:
Done.